package.xml

```diff
@@ -18,8 +18,6 @@
 
   <run_depend>nao_description</run_depend>
   <run_depend>joint_state_publisher</run_depend>
-  <run_depend>moveit_fake_controller_manager</run_depend>
-  <run_depend>moveit_simple_controller_manager</run_depend>
```

Comment:
why did you remove the controller dependencies ? can you run demo.launch without those packages installed ?